### DISCUSSION TITLE
Who recorded the title song for 2021 Bond film No Time To Die?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#213] [Galaxy Trivia] [answered]
**Hint:** Bad Guy

**Comment by Faiqlatheef** [comment, marked as answer]
The title song for the 2021 Bond film No Time To Die was recorded by the American singer-songwriter Billie Eilish. The song was co-written by Eilish and her brother Finneas O'Connell, and was released in February 2020, ahead of the film's original release date in April 2020. The song received critical acclaim and was a commercial success, reaching the top 10 in several countries, including the UK and the US. It also won the Grammy Award for Best Song Written for Visual Media in 2021.
